backend/src/test/java/org/donggle/backend/domain/parser/notion/NotionParserTest.java

```diff
@@ -0,0 +1,126 @@
+package org.donggle.backend.domain.parser.notion;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.donggle.backend.application.service.notion.NotionBlockNode;
+import org.donggle.backend.domain.writing.BlockType;
+import org.donggle.backend.domain.writing.Style;
+import org.donggle.backend.domain.writing.StyleRange;
+import org.donggle.backend.domain.writing.StyleType;
+import org.donggle.backend.domain.writing.content.CodeBlockContent;
+import org.donggle.backend.domain.writing.content.Content;
+import org.donggle.backend.domain.writing.content.Depth;
+import org.donggle.backend.domain.writing.content.ImageCaption;
+import org.donggle.backend.domain.writing.content.ImageContent;
+import org.donggle.backend.domain.writing.content.ImageUrl;
+import org.donggle.backend.domain.writing.content.Language;
+import org.donggle.backend.domain.writing.content.NormalContent;
+import org.donggle.backend.domain.writing.content.RawText;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class NotionParserTest {
+    @Test
+    @DisplayName("Paragraph타입 BlockNode로부터 NormalContent를 생성한다.")
+    void createNormalContentFromBlockNode() {
+        //given
+        final JsonNode jsonNode = NotionBlockJsonBuilder.buildJsonBody("paragraph", false);
+        final NotionBlockNode notionBlockNode = new NotionBlockNode(jsonNode, 0);
+        final NotionParser notionParser = new NotionParser();
+        final List<NotionBlockNode> notionBlockNodes = List.of(new NotionBlockNode(jsonNode, 0));
+        //when
+        final Content contents = notionParser.parseBody(notionBlockNodes).get(0);
+        //then
+        final NotionNormalBlockParser blockParser = DefaultBlockParser.from(notionBlockNode);
+        final String rawText = blockParser.parseRawText();
+        final List<Style> styles = blockParser.parseStyles();
+        final NormalContent expected = new NormalContent(Depth.from(0), BlockType.PARAGRAPH, RawText.from(rawText), styles);
+        assertThat(contents)
+                .usingRecursiveComparison()
+                .ignoringFields("id", "createdAt", "updatedAt", "styles")
+                .isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("Code타입 BlockNode로부터 CodeBlockContent를 생성한다.")
+    void createCodeBlockContentFromBlockNode() {
+        //given
+        final JsonNode jsonNode = NotionBlockJsonBuilder.buildJsonBody("code", false);
+        final NotionBlockNode notionBlockNode = new NotionBlockNode(jsonNode, 0);
+        final NotionParser notionParser = new NotionParser();
```

Comment:
해당 코드가 계속 반복되는 것 같은데 beforeEach로 빼주는건 어떨까요 ?

```suggestion
private NotionBlockNode notionBlockNode;
private NotionParser notionParser;

@BeforeEach
void setUp(){
        notionBlockNode = new NotionBlockNode(jsonNode, 0);
        notionParser = new NotionParser();
}
```